src/seedu/addressbook/parser/Parser.java

```diff
@@ -1,28 +1,14 @@
 package seedu.addressbook.parser;
 
-import static seedu.addressbook.common.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
-import static seedu.addressbook.common.Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX;
+import seedu.addressbook.commands.*;
```

Comment:
Minor comment.
Imported classes should always be listed explicitly.

Please look at https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements